src/python/pants/backend/python/rules/pytest_coverage.py

```diff
@@ -212,8 +214,21 @@ def source_root_stripped_source_and_source_root(file_name: str) -> Tuple[str, st
 
 
 class ReportType(Enum):
-    XML = "xml"
-    HTML = "html"
+    CONSOLE = ("console", "report")
+    XML = ("xml", None)
+    HTML = ("html", None)
+
+    _report_name: str
+
+    def __new__(cls, value: str, report_name: Optional[str] = None):
```

Comment:
I think this returns `-> "ReportType"`?

Reply:
Yeah. Self types are pretty unsatisfying right now. Fixed.

Reply:
Agreed. `from __future__ import annotations` will make things a little better once we drop Py36.